docs/integrations/microsoft-azure/azure-container-instances.md

```diff
@@ -0,0 +1,210 @@
+---
+id: azure-container-instances
+title: Azure Container Instances
+description: Learn about the Sumo Logic collection process for the Azure Container Instances service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-container-instances.png')} alt="Thumbnail icon" width="50"/>
+
+[Azure Container Instances](https://learn.microsoft.com/en-us/azure/container-instances/) is a fully managed serverless container service that enables you to deploy and manage containers in Azure without the need for virtual machines. It is ideal for event-driven applications, microservices, and batch processing workloads.
+
+## Log and metric types
+
+For Azure Container Instances, you can collect the following logs and metrics:
+
+* **Resource Logs**. Capture container creation, execution, and failure logs. The schema for resource logs is described [here](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-monitor#resource-logs).
+* **Metrics**. Metrics for Azure Container Instances are in the following namespaces:
+  * [Microsoft.ContainerInstance/containerGroups](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-containerinstance-containergroups-metrics)
+
+For more information on supported dimensions, refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-monitor#metrics).
+
+## Setup
+
+Azure service sends monitoring data to Azure Monitor, which can then [stream data to an event hub](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/stream-monitoring-data-event-hubs). Sumo Logic supports:
```

Comment:
This whole setup section will change since we are using fluentbit and azure beta metrics source